pkg/cloudprovider/fake/api.go

```diff
@@ -20,23 +20,35 @@ import (
 	"strings"
 
 	"github.com/Pallinder/go-randomdata"
+	"github.com/awslabs/karpenter/pkg/apis/provisioning/v1alpha1"
 	"github.com/awslabs/karpenter/pkg/cloudprovider"
+
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"knative.dev/pkg/apis"
 )
 
-type Capacity struct {
-}
-type Termination struct {
+var (
+	NotImplementedError = fmt.Errorf("provider is not implemented. Are you running the correct release for your cloud provider?")
+)
+
+type API struct {
```

Comment:
I wouldn't make this name change for similar reasons. This looks like a bad merge to me. I removed the NotImplementedError (which isn't used anymore) in a previous PR.